core/src/test/java/org/apache/accumulo/core/conf/SiteConfigurationTest.java

```diff
@@ -61,7 +61,9 @@ public void testDefault() {
     assertEquals("DEFAULT", conf.get(Property.INSTANCE_SECRET));
     assertEquals("", conf.get(Property.INSTANCE_VOLUMES));
     assertEquals("120s", conf.get(Property.GENERAL_RPC_TIMEOUT));
-    assertEquals("1G", conf.get(Property.TSERV_WALOG_MAX_SIZE));
+    @SuppressWarnings("deprecation")
+    Property deprecatedProp = Property.TSERV_WALOG_MAX_SIZE;
+    assertEquals("1G", conf.get(conf.resolve(Property.TSERV_WAL_MAX_SIZE, deprecatedProp)));
```

Comment:
Since this test is just testing the default values for a bunch of props not set in a config file, it'd be fine to just use the new prop, without calling `conf.resolve`.